command/rootchain/fund/fund.go

```diff
@@ -109,18 +119,42 @@ func runCommand(cmd *cobra.Command, _ []string) {
 
 		receipt, err := txRelayer.SendTransactionLocal(txn)
 		if err != nil {
-			outputter.SetError(err)
+			errCh <- err
 
 			return
 		}
 
-		resList[i] = &result{
+		lock.Lock()
+
+		resList = append(resList, &result{
 			ValidatorAddr: validatorAcc,
 			TxHash:        types.Hash(receipt.TransactionHash),
-		}
+		})
+
+		lock.Unlock()
 	}
 
-	outputter.SetCommandResult(resList)
+	for _, params := range paramsList {
+		wg.Add(1)
+
+		go sendFundTx(params)
```

Comment:
Couldn't we use `errGroup` https://pkg.go.dev/golang.org/x/sync/errgroup?

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1399/commits/617d795bdcc179c215a3788bc1c1f5de7c2c8492